src/components/GenerativeDirectAnswer.tsx

```diff
@@ -0,0 +1,167 @@
+import { 
+  GenerativeDirectAnswerResponse, 
+  useSearchActions, 
+  useSearchState, 
+  Result
+} from '@yext/search-headless-react';
+import { useComposedCssClasses } from '../hooks';
+import { executeGenerativeDirectAnswer } from '../utils/search-operations';
+import ReactMarkdown from "react-markdown";
+import React from 'react';
+
+/**
+ * The CSS class interface used for {@link GenerativeDirectAnswer}.
+ *
+ * @public
+ */
+export interface GenerativeDirectAnswerCssClasses {
+  container?: string,
+  header?: string,
+  answerText?: string,
+  divider?: string,
+  citationsContainer?: string,
+  citation?: string,
+  citationTitle?: string,
+  citationSnippet?: string
+}
+
+const builtInCssClasses: Readonly<GenerativeDirectAnswerCssClasses> = {
+  container: 'p-6 border border-gray-200 rounded-lg shadow-sm',
+  header: 'text-xl',
+  answerText: 'mt-4',
+  divider: 'border-b border-gray-200 w-full pb-6 mb-6',
+  citationsContainer: 'mt-4 flex overflow-x-auto gap-4',
+  citation: 'p-4 border border-gray-200 rounded-lg shadow-sm bg-slate-100 flex flex-col grow-0 shrink-0 basis-64 text-sm text-neutral',
+  citationTitle: 'font-bold',
+  citationSnippet: 'line-clamp-2 text-ellipsis break-words'
```

Comment:
per discussion offline, we should update the overflow-x css for the parent div to achieve the desired result